compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/Scrollbar.desktop.kt

```diff
@@ -379,8 +379,9 @@ internal fun <T> OldOrNewScrollbar(
     val adapter = remember(oldOrNewAdapter, containerSize){
         newScrollbarAdapterFactory(oldOrNewAdapter, containerSize)
     }
+    val coroutineScope = rememberCoroutineScope()
     val sliderAdapter = remember(adapter, containerSize, minimalHeight, reverseLayout, isVertical) {
```

Comment:
Better put `coroutineScope` in the `remember` keys too, just in case...

Reply:
done